src/utils/texture.js

```diff
@@ -96,7 +96,7 @@ function loadVideo (material, data, src) {
   textureCache[calculateVideoCacheHash(videoEl)] = Promise.resolve(texture, videoEl);
   handleVideoTextureLoaded(texture, videoEl);
 
-  function handleVideoTextureLoaded (texture, videoEl) {
+  function handleVideoTextureLoaded (texture) {
```

Comment:
I don't think this will work. Other textures waiting on the promise need to know about the `videoEl`. Maybe we should try renaming this parameter? `resolvedVideoEl`.